client/transaction-pool/graph/src/validated_pool.rs

```diff
@@ -560,7 +562,7 @@ impl<B: ChainApi> ValidatedPool<B> {
 
 	/// Notify all watchers that transactions in the block with hash have been finalized
 	pub async fn on_block_finalized(&self, block_hash: BlockHash<B>) -> Result<(), B::Error> {
-		debug!(target: "txpool", "Attempting to notify watchers of finalization for {}", block_hash);
```

Comment:
This is particular annoying useless spam

moved `debug!` instead to the place where some action actually take (if at all)